### PR TITLE
Bump minpy to 3.8

### PR DESCRIPTION
**Pull request by bryevdv**

### CODE REVIEW

File: conda/conda-build/conda_build_config.yaml [hunk 1]
```diff
@@ -2,9 +2,8 @@
 # so use an empty string "" to denote a falsey value
 gpu_enabled:
   - true
-  - 
+  -
```
Comment (bryevdv) [inline conda/conda-build/conda_build_config.yaml:5]: This was already here but not sure why, can it be removed entirely?

Comment (manopapad) [reply conda/conda-build/conda_build_config.yaml:5]: ^ @marcinz @m3vaz ?

**Comment by manopapad** [comment]
I also updated some python references in the README

File: conda/conda-build/conda_build_config.yaml (hunk 1 above)
Comment (m3vaz) [reply conda/conda-build/conda_build_config.yaml:5]: Line 1 and 2:
```
# Conda only accepts strings as variant configs
# so use an empty string "" to denote a falsey value
```

Comment (bryevdv) [reply conda/conda-build/conda_build_config.yaml:5]: The more you know :shrug: thanks @m3vaz

**Review by m3vaz — CHANGES_REQUESTED** [review]
The falsey value is required to trigger the cpu only spec. Please do not remove it from the recipe.

**Comment by bryevdv** [comment]
@m3vaz  https://github.com/nv-legate/legate.core/pull/220/commits/8fa0b0a042557367e8fca901345fae9bf71b92e6

**Review by manopapad — APPROVED** [review]

**Review by m3vaz — APPROVED** [review]

**Comment by m3vaz** [comment]
LGTM, Please tag me when you have the cunumeric PR ready as well

**Pull request merged**